app/controllers/admin/topics_controller.rb

```diff
@@ -312,6 +312,43 @@ def assign_team
     end
   end
 
+  def split_topic
+    parent_topic = Topic.find(params[:topic_id])
+    parent_post = Post.find(params[:post_id])
+
+    @topic = Topic.new(
+      name: t('new_discussion_topic_title', original_topic: parent_topic.name, default: "Split from #{parent_topic.name}"),
+      user: parent_topic.user,
+      forum_id: 1,
+    )
```

Comment:
I was testing this out and it looks like we also need to explicitly set the `private` flag to `true`.  However, I was also thinking about this and maybe we should just set the `forum_id` and `private` to the `parent_topic` settings.  That way it would work correctly for private tickets, and also for public posts (just in case).